.github/workflows/backend.yml

```diff
@@ -51,6 +52,8 @@ jobs:
         os: ["ubuntu-latest", "windows-latest"]
         test-type: ["not flaky", "flaky"]
         python-version: ["3.8"]
+        exclude:
+          - os: ${{ github.event_name == 'pull_request' && contains( github.event.pull_request.labels.*.name, 'windows-tests') && 'dummy' || 'windows-latest' }}
```

Comment:
This is a cool way to conditionally exclude cases from the matrix. See https://stackoverflow.com/a/73822998